js/mobile-menu.js

```diff
@@ -0,0 +1,23 @@
+const hamburger = document.querySelector('.hamburger');
+const overlay = document.querySelector('.overlay');
+const menu = document.querySelector('.menu');
+const menuLink = document.querySelector('.menu-link');
+const logo = document.querySelector('.logo');
+
+    hamburger.addEventListener('click', ()=>{
+    hamburger.classList.toggle('active');
+    overlay.classList.toggle('active');
+    menu.classList.toggle('active');
+    logo.classList.toggle('active');
+    });
+
+    menuLink.forEach((link)=>{
+    link.addEventListener('click', ()=>{
+    hamburger.classList.remove('active');
+    menu.classList.remove('active');
+    logo.classList.remove('active');
+    overlay.classList.remove('active');
+
+});
+});
+
```

Comment:
- [ ] When I click on nav links it takes me to the right section but the mobile menu still appears on the page and when I click on any nav link it supposes to do both closes the mobile menu and takes me to the correct section, So please fix that.

![chrome-capture-2022-4-17](https://user-images.githubusercontent.com/68796232/168851972-51632a23-1c8e-49d3-84c4-3b75b1939efe.gif)